uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/ScalebarController.qml

```diff
@@ -26,7 +26,9 @@ import QtQml 2.12
 
     This controller object handles the Scalebar calculations for a Scalebar's width
     and display units, based on a given mapview and owning scalebar's bounds.
+    \note This controller has not been implemented yet for QML.
  */
+//todo: once implemented, add this file as a \l to the docs on scalebar.qml controller property. remove the \note above
```

Comment:
Just 'cos I'm paranoid: this note between the doc and the implementation doesn't affect the output, does it?

Reply:
no, this comment is not printed in the docs

Reply:
But re-reading your question, did you mean the line 29, `\note This controller has not been implemented yet for QML`? This one is outputted in the docs

Reply:
Just needed to make sure this comment didn't affect the above docs! Sounds good!